conf/base.config

```diff
@@ -10,25 +10,42 @@
  */
 
 process {
+   cpus = { check_max( 2, 'cpus' ) }
+   memory = { check_max( 8.GB * task.attempt, 'memory' ) }
+   time = { check_max( 4.h * task.attempt, 'time' ) }
 
-  // TODO nf-core: Check the defaults for all processes
-  cpus = { check_max( 1 * task.attempt, 'cpus' ) }
-  memory = { check_max( 8.GB * task.attempt, 'memory' ) }
-  time = { check_max( 2.h * task.attempt, 'time' ) }
+   errorStrategy = { task.exitStatus in [143,137,104,134,139] ? 'retry' : 'terminate' }
+   maxRetries = 1
+   maxErrors = '-1'
 
-  errorStrategy = { task.exitStatus in [143,137,104,134,139] ? 'retry' : 'finish' }
-  maxRetries = 1
-  maxErrors = '-1'
-
-  // Process-specific resource requirements
-  // TODO nf-core: Customise requirements for specific processes.
-  // See https://www.nextflow.io/docs/latest/config.html#config-process-selectors
+   // Process-specific resource requirements
+   withLabel: low_memory {
```

Comment:
Be good if we can name these according to the pipeline [template](https://github.com/nf-core/tools/blob/f14c7a5692c96896c2ed1b730bc355093dca2247/nf_core/pipeline-template/%7B%7Bcookiecutter.name_noslash%7D%7D/conf/base.config#L29)